### PR TITLE
Fix erblint failures

### PR DESCRIPTION
**Pull request by djuber**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

- a closing quote on the `aria-haspopup="true ` caused some issues when
running erblint (lots of keywords were assumed to be strings, and vice versa)

- a warning about Lint/FloatComparison raised about casting the
rating_level value to_f, I chose to convert the float to integer and
compare to the value instead.

I suspect the lint rule (from git hooks) didn't run in the most recent changes to this file.

## Related Tickets & Documents

https://github.com/ludwiczakpawel/dev.to/commit/38214380ddb3445a2cca0ce04d222a8138dac2ce the missing closing quote was introduced a few months ago.

The float cast (which may be a new lint rule triggering?) was added in https://github.com/forem/forem/pull/7777 so has been there for over a year (and seems like it wasn't as unreliable for comparisons as the lint rule suggested).

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://docs.forem.com/frontend/accessibility)._

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://admin.forem.com/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [x] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/views/moderations/actions_panel.html.erb [hunk 1]
```diff
@@ -141,7 +141,7 @@
         value="<%= rating_level[0] %>"
         name="rating_vote_<%= rating_level[0] %>"
         id="js__rating__vote__<%= rating_level[0] %>"
-        class="level-rating-button <%= " selected" if @rating_vote&.rating == rating_level[0].to_f %>"
+        class="level-rating-button <%= " selected" if @rating_vote&.rating&.to_i == rating_level[0] %>"
```
Comment (djuber) [inline app/views/moderations/actions_panel.html.erb:144]: I'm least certain of this change -  I'm not opposed to ignoring the rubocop rule being called by the linter here.

**Review by nickytonline — APPROVED** [review]
Noice 🚢

File: app/views/moderations/actions_panel.html.erb (hunk 1 above)
Comment (djuber) [reply app/views/moderations/actions_panel.html.erb:144]: I'm also wondering whether `rating.round(0)` were the better option (I can never remember if to_i is truncation or not).

Comment (nickytonline) [reply app/views/moderations/actions_panel.html.erb:144]: Can we add a custom Rubocop rule once we decide which one is the way to go?

Comment (djuber) [reply app/views/moderations/actions_panel.html.erb]: I don't want to be the guy that mentions "types", but if think the rule would be "when converting a float to an integer, you will normally want to call round, not to_i", while ignoring the totally expected behavior of parsing strings as integers. I'd be  pleasantly surprised if inferring the type of a variable during static analysis were easy. 

Good

     "123".to_i => 123
     (123.6).round(0) => 124
     123.to_i => 123
     123.round(0) => 123 (weird looking, but correct)

Suspicious (not sure that it's wrong if it's the desired behavior, it usually isn't).

     (123.6).to_i => 123

**Review by citizen428 — APPROVED** [review]

**Review by rhymes — APPROVED** [review]
🔥

**Pull request merged**